### PR TITLE
Add Japanese Translation

### PR DESCRIPTION
**Pull request by vargaspeniel**
Adds complete Japanese translation for both the GUI and CLI texts.

### CODE REVIEW

File: po/ja.po [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Max memory usage (MB)"
+msgstr "最大メモリ使用 (MB)"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:229
+msgid "Max text item length"
+msgstr "最大テキスト項目の長さ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:236
+msgid "Min text item length"
+msgstr "最小テキスト項目の長さ"
```
Comment (ryonakano) [inline po/ja.po:113]:
```suggestion
msgstr "テキストの最小の長さ"
```

Same as the previous one.

File: po/ja.po [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:297
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:77
+msgid "Sync the primary selection to the clipboard"
+msgstr "一次選択をクリップボードに同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:304
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:78
+msgid "Upload the active item to a pastebin service"
+msgstr "ペーストビン・サービスにアクティブな項目をアップデートする"
```
Comment (ryonakano) [inline po/ja.po:148]:
```suggestion
msgstr "Pastebin にアクティブな項目をアップロードする"
```

"a pastebin service" is a name of service, so we may want to keep it as alphabets. "アップデート" is a translation of "update", not "upload". "upload" should be translated as "アップロード". They sound and look similar, but different words :sweat_smile:

File: po/ja.po [hunk 3]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:319
+#: src/libgpaste/ui/gpaste-ui-window.c:311 src/client/gpaste-client.c:789
+msgid "Couldn't connect to GPaste daemon"
+msgstr "GPaste デーモンに接続できませんでした"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:334
+msgid "General behaviour"
+msgstr "一般行動"
```
Comment (ryonakano) [inline po/ja.po:157]:
```suggestion
msgstr "挙動全般"
```

"behaviour" usually means "挙動"―how the app works or behaves.

File: po/ja.po [hunk 4]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:335
+msgid "History settings"
+msgstr "履歴設定"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:336
+msgid "Keyboard shortcuts"
+msgstr "キーボード・ショートカット"
```
Comment (ryonakano) [inline po/ja.po:165]:
```suggestion
msgstr "キーボードショートカット"
```

Yours IS correct, but IIRC this translation is used in other GNOME components.

File: po/ja.po [hunk 5]
```diff
@@ -0,0 +1,561 @@
...
+
+#. translators: This is the name of a multi-history management action
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:453
+#: src/libgpaste/ui/gpaste-ui-panel.c:297
+msgid "Switch to"
+msgstr "切り替える"
```
Comment (ryonakano) [inline po/ja.po:171]:
```suggestion
msgstr "切り替え"
```

File: po/ja.po [hunk 6]
```diff
@@ -0,0 +1,561 @@
...
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:458
+#: src/libgpaste/ui/gpaste-ui-delete-item.c:49
+#: src/libgpaste/ui/gpaste-ui-delete-history.c:25
+#: src/libgpaste/ui/gpaste-ui-delete-history.c:65
+msgid "Delete"
+msgstr "削除"
+
+#: src/libgpaste/ui/gpaste-ui-about.c:41
+msgid "About"
+msgstr "このプログラムについて"
```
Comment (ryonakano) [inline po/ja.po:183]:
```suggestion
msgstr "GPaste について"
```

Yes, yours is correct too, but it's common to translate this string as "\<name-of-app\> について" in other GNOME components.

File: po/ja.po [hunk 7]
```diff
@@ -0,0 +1,561 @@
...
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:45
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:118
+msgid "Edit"
+msgstr "編集"
+
+#. Translators: this is the translation for emptying the history
+#: src/libgpaste/ui/gpaste-ui-empty-history.c:65
+#: src/libgpaste/util/gpaste-util.c:410
+msgid "Empty"
+msgstr "履歴は空です"
```
Comment (ryonakano) [inline po/ja.po:213]:
```suggestion
msgstr "空にする"
```

See the above comment for translators:

> Translators: this is the translation for emptying the history

File: po/ja.po [hunk 8]
```diff
@@ -0,0 +1,561 @@
...
+msgid "New"
+msgstr "新規"
+
+#: src/libgpaste/ui/gpaste-ui-reexec.c:29
+msgid "Restart"
+msgstr "再起動"
+
+#: src/libgpaste/ui/gpaste-ui-reexec.c:29
+msgid "Do you really want to restart the daemon?"
+msgstr "本当に、デーモンを再起動しますか？"
```
Comment (ryonakano) [inline po/ja.po:238]:
```suggestion
msgstr "本当にデーモンを再起動しますか？"
```

I don't think we need `、` here :thinking:

File: po/ja.po [hunk 9]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Restart the daemon"
+msgstr "デーモンを再起動する"
+
+#: src/libgpaste/ui/gpaste-ui-search.c:26
+msgid "Search"
+msgstr "検索"
+
+#: src/libgpaste/ui/gpaste-ui-settings.c:32 src/ui/gpaste-ui.c:105
+msgid "GPaste Settings"
+msgstr "GPaste 設定"
```
Comment (ryonakano) [inline po/ja.po:250]:
```suggestion
msgstr "GPaste の設定"
```

"GPaste 設定" (in a word) sounds like a proper noun to me.

File: po/ja.po [hunk 10]
```diff
@@ -0,0 +1,561 @@
...
+msgid "General"
+msgstr "一般"
+
+#: src/libgpaste/ui/gpaste-ui-switch.c:54
+msgid "Stop"
+msgstr "停止"
+
+#: src/libgpaste/ui/gpaste-ui-switch.c:54
+msgid "Do you really want to stop tracking clipboard changes?"
+msgstr "本当に、クリップボードの変更の追跡を停止しますか？"
```
Comment (ryonakano) [inline po/ja.po:262]:
```suggestion
msgstr "本当にクリップボードの変更の追跡を停止しますか？"
```

File: po/ja.po [hunk 11]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/ui/gpaste-ui-upload-item.c:49
+msgid "Upload"
+msgstr "アップデート"
```
Comment (ryonakano) [inline po/ja.po:266]:
```suggestion
msgstr "アップロード"
```

File: po/ja.po [hunk 12]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/util/gpaste-util.c:410
+msgid "Do you really want to empty the history?"
+msgstr "本当に、履歴を空にしますか？"
```
Comment (ryonakano) [inline po/ja.po:270]:
```suggestion
msgstr "本当に履歴を空にしますか？"
```

File: po/ja.po [hunk 13]
```diff
@@ -0,0 +1,561 @@
...
+msgstr "利用可能な履歴を一覧表示する"
+
+#. Translators: help for gpaste add <text>
+#: src/client/gpaste-client.c:165
+msgid "text"
+msgstr "テキスト"
+
+#: src/client/gpaste-client.c:165
+msgid "set text to clipboard"
+msgstr "テキストをクリップボードに設定する"
```
Comment (ryonakano) [inline po/ja.po:335]:
```suggestion
msgstr "テキストをクリップボードに追加する"
```

"設定" sounds "settings", "preferences", or "configurations". I guess it's OK to translate it as "add" here.

File: po/ja.po [hunk 14]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:167
+msgid "password"
+msgstr "パスワード"
+
+#: src/client/gpaste-client.c:167
+msgid "add the name - password couple to the clipboard"
+msgstr "クリップボードに名前・パスワードの一組を追加する"
```
Comment (ryonakano) [inline po/ja.po:343]:
```suggestion
msgstr "クリップボードに名前とパスワードの組み合わせを追加する"
```

File: po/ja.po [hunk 15]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste rename-password <old name> <new name>
+#: src/client/gpaste-client.c:169
+msgid "old name"
+msgstr "旧名"
```
Comment (ryonakano) [inline po/ja.po:348]:
```suggestion
msgstr "古い名前"
```

"旧" and "古" is the same meaning, but the former is rarely used nowadays and only in some special words like "旧街道" or "旧式".

File: po/ja.po [hunk 16]
```diff
@@ -0,0 +1,561 @@
...
+#. Translators: help for gpaste set-password <number> <name>
+#. Translators: help for gpaste delete <number>
+#: src/client/gpaste-client.c:171 src/client/gpaste-client.c:177
+#: src/client/gpaste-client.c:179 src/client/gpaste-client.c:181
+msgid "number"
+msgstr "数字"
+
+#: src/client/gpaste-client.c:171
+msgid "get the <number>th item from the history"
+msgstr "履歴から<number>番目の項目を取得する"
```
Comment (ryonakano) [inline po/ja.po:369]:
```suggestion
msgstr "履歴の <数字> 番目の項目を取得する"
```

You should translate this too :)

File: po/ja.po [hunk 17]
```diff
@@ -0,0 +1,561 @@
...
+msgid ""
+"replace the contents of the item matching <uuid> from the history with the "
+"provided one"
+msgstr "履歴の <uuid> に一致する項目の内容を提供されたもので置き換える"
+
+#: src/client/gpaste-client.c:177
+msgid ""
+"merge the <number>th items from the history and add put the result in the "
+"clipboard"
+msgstr "履歴から<number>番目の項目をマージして、結果をクリップボードに追加する"
```
Comment (ryonakano) [inline po/ja.po:391]:
```suggestion
msgstr "履歴の <数字> 番目の項目をマージして、結果をクリップボードに追加する"
```

File: po/ja.po [hunk 18]
```diff
@@ -0,0 +1,561 @@
...
+#. Translators: help for gpaste replace <number> <contents>
+#: src/client/gpaste-client.c:175
+msgid "contents"
+msgstr "内容"
+
+#: src/client/gpaste-client.c:175
+msgid ""
+"replace the contents of the item matching <uuid> from the history with the "
+"provided one"
+msgstr "履歴の <uuid> に一致する項目の内容を提供されたもので置き換える"
```
Comment (ryonakano) [inline po/ja.po:385]:
```suggestion
msgstr "履歴の <uuid> に一致する項目の内容を指定した項目で置き換える"
```

"提供" is literally OK, but sounds exaggerated.

File: po/ja.po [hunk 19]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:177
+msgid ""
+"merge the <number>th items from the history and add put the result in the "
+"clipboard"
+msgstr "履歴から<number>番目の項目をマージして、結果をクリップボードに追加する"
+
+#: src/client/gpaste-client.c:179
+msgid "set the <number>th item from the history as a password named <name>"
+msgstr "履歴から<number>番目の項目を<name>という名前のパスワードとして設定する"
```
Comment (ryonakano) [inline po/ja.po:395]:
```suggestion
msgstr "履歴の <数字> 番目の項目を <名前> という名前のパスワードとして設定する"
```

File: po/ja.po [hunk 20]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:181
+msgid "delete <number>th item of the history"
+msgstr "履歴から<number>番目の項目を削除する"
```
Comment (ryonakano) [inline po/ja.po:399]:
```suggestion
msgstr "履歴の <数字> 番目の項目を削除する"
```

File: po/ja.po [hunk 21]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:183
+msgid "delete the password <name> from the history"
+msgstr "履歴からパスワード<name>を削除する"
```
Comment (ryonakano) [inline po/ja.po:403]:
```suggestion
msgstr "履歴からパスワード <名前> を削除する"
```

File: po/ja.po [hunk 22]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste file <path>
+#: src/client/gpaste-client.c:185
+msgid "path"
+msgstr "パス"
+
+#: src/client/gpaste-client.c:185
+msgid "put the content of the file at <path> into the clipboard"
+msgstr "<path>にあるファイルの内容をクリップボードに入れる"
```
Comment (ryonakano) [inline po/ja.po:412]:
```suggestion
msgstr "<パス> にあるファイルの内容をクリップボードに追加"
```

File: po/ja.po [hunk 23]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for whatever | gpaste
+#: src/client/gpaste-client.c:187
+msgid "whatever"
+msgstr "プログラム"
```
Comment (ryonakano) [inline po/ja.po:417]: Nice translation :blush:

File: po/ja.po [hunk 24]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste stop
+#: src/client/gpaste-client.c:193
+msgid "stop tracking clipboard changes"
+msgstr "クリップボードの変更の追跡を停止する"
+
+#. Translators: help for gpaste quit
+#: src/client/gpaste-client.c:195
+msgid "alias for stop"
+msgstr "停止の別名"
```
Comment (ryonakano) [inline po/ja.po:441]:
```suggestion
msgstr "gpaste stop と同じ"
```

It's an alias for `gpaste stop` so `gpaste stop` and `gpaste quit` should have the same behavior. So we can translate it the same as the translation of the description of `gpaste stop` or something like this.

File: po/ja.po [hunk 25]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste show-history
+#: src/client/gpaste-client.c:203
+msgid "make the applet or extension display the history"
+msgstr "アプレットまたは拡張機能に履歴を表示させる"
+
+#. Translators: help for gpaste upload
+#: src/client/gpaste-client.c:205
+msgid "upload the item matching <uuid> to a pastebin service"
+msgstr "<uuid> に一致する項目をペーストビン・サービスにアップロードする"
```
Comment (ryonakano) [inline po/ja.po:466]:
```suggestion
msgstr "<uuid> に一致する項目を Pastebin にアップロードする"
```

File: po/ja.po [hunk 26]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste version
+#: src/client/gpaste-client.c:207
+msgid "display the version"
+msgstr "バージョン情報を表示する"
```
Comment (ryonakano) [inline po/ja.po:471]:
```suggestion
msgstr "バージョンを表示する"
```

In the next string you translate "デーモンのバージョンを表示する", not "デーモンのバージョン**情報**を表示する", so we can omit "情報" here too for consistency.

File: po/ja.po [hunk 27]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste daemon-version
+#: src/client/gpaste-client.c:209
+msgid "display the daemon version"
+msgstr "デーモンのバージョンを表示する"
+
+#. Translators: help for gpaste help
+#: src/client/gpaste-client.c:211
+msgid "display this help"
+msgstr "この使い方を表示する"
```
Comment (ryonakano) [inline po/ja.po:481]:
```suggestion
msgstr "ヘルプを表示する"
```

File: po/ja.po [hunk 28]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for gpaste about
+#: src/client/gpaste-client.c:213
+msgid "display the about dialog"
+msgstr "このプログラムについてを表示する"
```
Comment (ryonakano) [inline po/ja.po:486]:
```suggestion
msgstr "GPaste についてのダイアログを表示する"
```

"ダイアログ" means a dialog window: adding this word would make this translation more understandable.

File: po/ja.po [hunk 29]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:216
+#, c-format
+msgid "Display options:"
+msgstr "表示オプション:"
+
+#. Translators: help for --oneline
+#: src/client/gpaste-client.c:219
+msgid "display each item on only one line"
+msgstr "各項目を1行だけで表示する"
```
Comment (ryonakano) [inline po/ja.po:496]:
```suggestion
msgstr "各項目を一行に表示する"
```

File: po/ja.po [hunk 30]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for --raw
+#: src/client/gpaste-client.c:221
+msgid "display each item raw (without line numbers)"
+msgstr "各項目を未加工で表示する (行番号なし)"
```
Comment (ryonakano) [inline po/ja.po:501]:
```suggestion
msgstr "各項目を表示する (行番号なし)"
```

"raw" means that not doing any modification so "未加工" is correct, but I don't think we need to translate it explicitly :thinking:

File: po/ja.po [hunk 31]
```diff
@@ -0,0 +1,561 @@
...
+
+#. Translators: help for --reverse
+#: src/client/gpaste-client.c:223
+msgid "display the items in reverse order"
+msgstr "項目を逆順で表示する"
+
+#. Translators: help for --zero
+#: src/client/gpaste-client.c:225
+msgid "use a NUL character instead of a new line betweean each item"
+msgstr "各項目の間に改行の代わりにヌル文字を使用する"
```
Comment (ryonakano) [inline po/ja.po:511]:
```suggestion
msgstr "各項目の区切りに改行ではなくヌル文字を使用する"
```

File: po/ja.po [hunk 32]
```diff
@@ -0,0 +1,561 @@
...
+#. Translators: help for --separator <string>
+#: src/client/gpaste-client.c:231 src/client/gpaste-client.c:233
+msgid "string"
+msgstr "文字列"
+
+#: src/client/gpaste-client.c:231
+msgid ""
+"add the given decoration to the beginning and the end of each item before "
+"merging"
+msgstr "マージする前に各項目の最初と最後に与えられた装飾を追加する"
```
Comment (ryonakano) [inline po/ja.po:528]:
```suggestion
msgstr "マージする前に各項目の最初と最後に指定した文字列を追加する"
```

File: po/ja.po [hunk 33]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:233
+msgid "add the given separator between each item when merging"
+msgstr "マージする時に各項目の間に与えられた区切り記号を追加する"
```
Comment (ryonakano) [inline po/ja.po:532]:
```suggestion
msgstr "マージする時に各項目の間に指定された区切り記号を追加する"
```

File: po/ja.po [hunk 34]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/client/gpaste-client.c:330
+#, c-format
+msgid "Successfully reexecuted the daemon\n"
+msgstr "デーモンを正常に再実行しました\n"
+
+#: src/client/gpaste-client.c:479
+msgid "Cannot add non utf8 data as text."
+msgstr "非 UTF8 データをテキストとして追加する事はできません。"
```
Comment (ryonakano) [inline po/ja.po:541]:
```suggestion
msgstr "UTF-8 以外のデータはテキストとして追加できません。"
```

"〜することはできません" is more polite notation of "〜できません", but it's redundant in most cases and can be omitted.

File: po/ja.po [hunk 35]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/daemon/gpaste-daemon.c:33
+msgid "Stop signal received, exiting"
+msgstr "停止信号を受信し、終了します"
```
Comment (ryonakano) [inline po/ja.po:545]:
```suggestion
msgstr "停止シグナルを受信し、終了します"
```

"signal" is often translated in Katakana "シグナル" in programming or computer industries.

File: po/ja.po [hunk 36]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/daemon/gpaste-daemon.c:70
+msgid "Could not acquire DBus name."
+msgstr "DBus 名を取得できませんでした。"
+
+#: src/ui/gpaste-ui.c:106
+msgid "Keyboard Shortcuts"
+msgstr "キーボード・･ショートカット"
```
Comment (ryonakano) [inline po/ja.po:553]:
```suggestion
msgstr "キーボードショートカット"
```

File: po/ja.po [hunk 37]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Enable the gnome-shell extension"
+msgstr "GNOME Shell 拡張機能を有効化する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:137
+msgid "Sync the daemon state with the extension's one"
+msgstr "デーモンの状態を拡張機能の状態と同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:146
+msgid "Primary selection affects history"
+msgstr "一次選択は履歴に影響する"
```
Comment (sicklylife-jp) [inline po/ja.po:69]:
```suggestion
msgid "Primary selection affects history"
msgstr "プライマリー選択を履歴に反映させる"
```
Primary selection is called プライマリー選択 or プライマリセレクション in Japanese.

File: po/ja.po [hunk 38]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:152
+msgid "Synchronize clipboard with primary selection"
+msgstr "クリップボードを一次選択と同期する"
```
Comment (sicklylife-jp) [inline po/ja.po:73]:
```suggestion
msgid "Synchronize clipboard with primary selection"
msgstr "クリップボードをプライマリー選択と同期する"
```

File: po/ja.po [hunk 39]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Detect growing lines"
+msgstr "成長する線を検出する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:178
+msgid "Save history"
+msgstr "履歴を保存する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:201
+msgid "Max element size when displaying"
+msgstr "表示時の最大要素大きさ"
```
Comment (sicklylife-jp) [inline po/ja.po:93]:
```suggestion
msgid "Max element size when displaying"
msgstr "表示時の要素の最大サイズ"
```

File: po/ja.po [hunk 40]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:283
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:75
+msgid "Display the history"
+msgstr "履歴を表示する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:290
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:76
+msgid "Sync the clipboard to the primary selection"
+msgstr "クリップボードを一次選択に同期する"
```
Comment (sicklylife-jp) [inline po/ja.po:138]:
```suggestion
msgid "Sync the clipboard to the primary selection"
msgstr "クリップボードからプライマリー選択に同期する"
```

File: po/ja.po [hunk 41]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:297
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:77
+msgid "Sync the primary selection to the clipboard"
+msgstr "一次選択をクリップボードに同期する"
```
Comment (sicklylife-jp) [inline po/ja.po:143]:
```suggestion
msgid "Sync the primary selection to the clipboard"
msgstr "プライマリー選択からクリップボードに同期する"
```

File: po/ja.po [hunk 42]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:44
+#: src/libgpaste/ui/gpaste-ui-edit-item.c:46
+#: src/libgpaste/ui/gpaste-ui-new-item.c:32 src/libgpaste/util/gpaste-util.c:30
+msgid "Cancel"
+msgstr "キャンセル"
+
+#: src/libgpaste/ui/gpaste-ui-backup-history.c:47
+msgid "Under which name do you want to backup this history?"
+msgstr "この履歴をどの名前でバックアップしますか？"
```
Comment (sicklylife-jp) [inline po/ja.po:198]:
```suggestion
msgid "Under which name do you want to backup this history?"
msgstr "この履歴をどのような名前でバックアップしますか？"
```

File: po/ja.po [hunk 43]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Search"
+msgstr "検索"
+
+#: src/libgpaste/ui/gpaste-ui-settings.c:32 src/ui/gpaste-ui.c:105
+msgid "GPaste Settings"
+msgstr "GPaste 設定"
+
+#: src/libgpaste/ui/gpaste-ui-shortcuts-window.c:64
+msgid "General"
+msgstr "一般"
```
Comment (sicklylife-jp) [inline po/ja.po:254]:
```suggestion
msgid "General"
msgstr "全般"
```

File: po/ja.po [hunk 44]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Do you really want to empty the history?"
+msgstr "本当に、履歴を空にしますか？"
+
+#: src/libgpaste/util/gpaste-util.c:709
+msgid "Could not create history dir"
+msgstr "履歴ディレクトリを作成できませんでした"
+
+#: src/client/gpaste-client.c:137 src/client/gpaste-client.c:425
+msgid "Couldn't spawn"
+msgstr "生成できませんでした"
```
Comment (sicklylife-jp) [inline po/ja.po:278]:
```suggestion
msgid "Couldn't spawn"
msgstr "spawn できませんでした"
```

File: po/ja.po [hunk 45]
```diff
@@ -0,0 +1,561 @@
...
+"X-Generator: Poedit 2.2.4\n"
+"Plural-Forms: nplurals=1; plural=0;\n"
+
+#: src/libgpaste/gpaste-macros.h:98
+msgid "Failed to register the gtk application"
+msgstr "GTK アプリケーションを登録するのを失敗しました"
+
+#: src/libgpaste/gpaste-macros.h:110
+msgid "is already running."
+msgstr "既に実行しています。"
```
Comment (sicklylife-jp) [inline po/ja.po:27]:
```suggestion
msgid "is already running."
msgstr "はすでに実行しています。"
```
'は' is missing.

[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:27]: I see, thank you for the information.

File: po/ja.po [hunk 46]
```diff
@@ -0,0 +1,561 @@
...
+msgstr "[ファイル] "
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:116
+#: src/libgpaste/ui/gpaste-ui-switch.c:92
+msgid "Track clipboard changes"
+msgstr "クリップボードの変更を追跡する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:122
+msgid "Close UI on select"
+msgstr "選択際に UI を閉じる"
```
[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:57]: Sorry, totally forgot to type "〜する" when translating 😂. Thank you for the correction.

File: po/ja.po (hunk 37 above)
Comment (vargaspeniel) [reply po/ja.po:69]: Thank you. 😀

File: po/ja.po [hunk 47]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Images support"
+msgstr "画像対応"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:165
+msgid "Trim items"
+msgstr "項目をトリムする"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:171
+msgid "Detect growing lines"
+msgstr "成長する線を検出する"
```
[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:85]: This was one of the translations I was having trouble trying to understand its context and meaning in the program, thank you for the suggestion.

File: po/ja.po (hunk 2 above)
Comment (vargaspeniel) [reply po/ja.po:148]: まじですか！？完全に言葉を間違ったのは気づきませんでした。😱 😂 修正、本当にありがとうございます。
Really!? I totally did not realize that I had mistaken the words. Thank you for the correction.

File: po/ja.po (hunk 6 above)
Comment (vargaspeniel) [reply po/ja.po:183]: Thanks. Was not sure how to translate this.

File: po/ja.po (hunk 7 above)
Comment (vargaspeniel) [reply po/ja.po:213]: Thanks. I got a little confused with this one.

File: po/ja.po (hunk 13 above)
Comment (vargaspeniel) [reply po/ja.po:335]: I see, thanks.

File: po/ja.po (hunk 15 above)
Comment (vargaspeniel) [reply po/ja.po:348]: Interesting, thank you for the information.

File: po/ja.po (hunk 16 above)
Comment (vargaspeniel) [reply po/ja.po:369]: Thanks. Was not sure if I should translate this or not, thought it was a program variable or parameter.

File: po/ja.po (hunk 23 above)
Comment (vargaspeniel) [reply po/ja.po:417]: Thank you. 😄

File: po/ja.po (hunk 24 above)
Comment (vargaspeniel) [reply po/ja.po:441]: I see. Thanks for the explanation.

File: po/ja.po (hunk 34 above)
Comment (vargaspeniel) [reply po/ja.po:541]: Thank you for the explanation.

File: po/ja.po (hunk 35 above)
Comment (vargaspeniel) [reply po/ja.po:545]: Thanks. Was not sure how to translate this one too.

File: po/ja.po (hunk 45 above)
Comment (vargaspeniel) [reply po/ja.po:27]: Thank you.

File: po/ja.po [hunk 48]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Primary selection affects history"
+msgstr "一次選択は履歴に影響する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:152
+msgid "Synchronize clipboard with primary selection"
+msgstr "クリップボードを一次選択と同期する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:159
+msgid "Images support"
+msgstr "画像対応"
```
[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:77]: Thanks. Wasn't really sure of the context in this one.

File: po/ja.po [hunk 49]
```diff
@@ -0,0 +1,561 @@
...
+msgid "Save history"
+msgstr "履歴を保存する"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:201
+msgid "Max element size when displaying"
+msgstr "表示時の最大要素大きさ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:208
+msgid "Max displayed history size"
+msgstr "最大表示履歴大きさ"
```
[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:97]: I see, thanks for the suggestion. Got a little confused with this one.

File: po/ja.po [hunk 50]
```diff
@@ -0,0 +1,561 @@
...
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:215
+msgid "Max history size"
+msgstr "最大履歴大きさ"
+
+#: src/libgpaste/settings-ui/gpaste-settings-ui-stack.c:222
+msgid "Max memory usage (MB)"
+msgstr "最大メモリ使用 (MB)"
```
[the remark this answers is not in the document]
Comment (vargaspeniel) [reply po/ja.po:105]: Totally forgot to add "量", thanks for the correction.

File: po/ja.po (hunk 4 above)
Comment (vargaspeniel) [reply po/ja.po:165]: I see.

**Comment by vargaspeniel** [comment]
@ryonakano さん @sicklylife-jp さん、こんにちは。

翻訳の提案や丁寧な説明と OSS での翻訳の助言も、誠にありがとうございます。とても勉強になりましたよ。知っての通り、私は日本人ではないのでまだ知らない単語や用語が多くて一人で本格的に OSS を翻訳するのも始めてです。翻訳していた時に原文の文脈を少し分からなっかたので、正しい翻訳に困りました。けれど、皆さんのお陰様で当時に分からなっかた事を分かってきた気がします。

改めて、ありがとうございます。そして、これからも宜しくお願いします。

Thank you very much for your translation suggestions, careful explanations, and advice when translating OSS. They are very useful and informative.

**Comment by Keruspe** [comment]
Thanks a lot !

**Comment by vargaspeniel** [comment]
Your welcome. :smile:

**Comment by vargaspeniel** [comment]
You're welcome. :smile:

**Pull request merged**